skypeforlinux: 8.34.0.78 -> 8.42.76.54

r-ryantm opened the pull request:
Semi-automatic update generated by https://github.com/ryantm/nixpkgs-update tools. This update was made based on information from https://repology.org/metapackage/skypeforlinux/versions.



meta.description for skypeforlinux is: '"Linux client for skype"'.


<details>
<summary>
Checks done (click to expand)
</summary>

- built on NixOS
- /bin/sh passed the binary check.
- 1 of 1 passed binary check by having a zero exit code.
- 0 of 1 passed binary check by having the new version present in output.
- found 8.42.76.54 with grep in /nix/store/51jqpi7x2a467kkqkc4cg4mdfr4qr094-skypeforlinux-8.42.76.54
- directory tree listing: https://gist.github.com/f57c858d3b0438477d9a794276512efd
- du listing: https://gist.github.com/595b1dcd0d2a98b265162cafcbe9fea6

</details>
<details>
<summary>
Rebuild report (if merged into master) (click to expand)
</summary>

1 total rebuild path(s)

1 package rebuild(s)

1 x86_64-linux rebuild(s)
0 i686-linux rebuild(s)
0 x86_64-darwin rebuild(s)
0 aarch64-linux rebuild(s)


First fifty rebuilds by attrpath
skypeforlinux

</details>

<details>
<summary>
Instructions to test this update (click to expand)
</summary>

Either download from Cachix:
```
nix-store -r /nix/store/51jqpi7x2a467kkqkc4cg4mdfr4qr094-skypeforlinux-8.42.76.54 \
  --option binary-caches 'https://cache.nixos.org/ https://r-ryantm.cachix.org/' \
  --option trusted-public-keys '
  r-ryantm.cachix.org-1:gkUbLkouDAyvBdpBX0JOdIiD2/DP1ldF3Z3Y6Gqcc4c=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(r-ryantm's Cachix cache is only trusted for this store-path realization.)

Or, build yourself:
```
nix-build -A skypeforlinux https://github.com/r-ryantm/nixpkgs/archive/1a714f16a44f854fcc02bf160acf3879474e0625.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/51jqpi7x2a467kkqkc4cg4mdfr4qr094-skypeforlinux-8.42.76.54
ls -la /nix/store/51jqpi7x2a467kkqkc4cg4mdfr4qr094-skypeforlinux-8.42.76.54/bin
```


</details>
<br/>


cc @panaeon @jraygauthier for testing.

r-ryantm committed:
skypeforlinux: 8.34.0.78 -> 8.42.76.54

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/skypeforlinux/versions

endgame commented:
I built this with:

```nix-build -E 'with import (builtins.fetchTarball { url = "https://github.com/r-ryantm/nixpkgs/archive/auto-update/skypeforlinux.tar.gz"; }) { config.allowUnfree = true; }; pkgs.skypeforlinux'
```

And then tried to start it with `./result/bin/skypeforlinux` and got
```./result/bin/skypeforlinux: error while loading shared libraries: libatk-bridge-2.0.so.0: cannot open shared object file: No such file or directory
```

Am I doing something wrong?

ryantm commented:
@endgame I get the same error message. It looks like a missing dependency that was added in the new version.

ryantm committed:
skypeforlinux: add missing dynamically linked library at-spi2-atk

The error message fixed was:

./result/bin/skypeforlinux: error while loading shared libraries:
libatk-bridge-2.0.so.0: cannot open shared object file: No such file
or directory

ryantm commented:
@endgame Can you test it again? It comes up now, but I don't have a Skype account to try.

endgame commented:
I have successfully built it and it runs. I logged in and tested camera but not mic.
